Repository/Models/Director.cs

```diff
@@ -7,7 +7,14 @@ namespace Repository.Models
 {
     public partial class Director
```

Comment:
porpose

Reply:
That is a database model, created by the database-first scaffolding tool.